spec/spec_helper_acceptance.rb

```diff
@@ -6,7 +6,7 @@
 
 def add_test_user
```

Comment:
this does not seem to follow the litmus template for spec_helper_acceptance. These helpers should be in spec_helper_acceptance_local.rb

Reply:
It is consistent with the documentation https://github.com/puppetlabs/puppet_litmus/wiki/Converting-a-module-to-use-Litmus#specspec_helper_acceptancerb

Reply:
Also, if I may ask, why are we adding "local" to the end of the file? That seems to imply that it's only used for localhost acceptance testing which isn't the case.

Reply:
@ThoughtCrhyme If you look at any of the modules, the _local.rb files denote local differences to the templated ruby files. hence https://github.com/puppetlabs/puppetlabs-apt/blob/master/spec/spec_helper_local.rb https://github.com/puppetlabs/puppetlabs-postgresql/blob/master/spec/spec_helper_local.rb for unit tests. https://github.com/puppetlabs/puppetlabs-postgresql/blob/master/spec/spec_helper_acceptance_local.rb and https://github.com/puppetlabs/puppetlabs-apt/blob/master/spec/spec_helper_acceptance_local.rb for non standard functions. If you feel this is unclear please update the wiki. 

The wiki states to use the template provided.